### PR TITLE
Update gdb & Victor's guide links, and fix voters link ignore

### LINKED ISSUE #1044: Victor Stinner’s guide has broken link in documentation
# Description:
while going through the python's developer guide for [window's installation and setup](https://devguide.python.org/getting-started/setup-building/#windows) I encountered a broken link for [Victor Stinner's guide](https://cpython-core-tutorial.readthedocs.io/en/latest/build_cpython_windows.html) and even the link on the error page is broken for [_Back to the front page of cpython-core-tutorial.readthedocs.io_](https://cpython-core-tutorial.readthedocs.io/) 

## Expected behavior:
The hyperlink should have taken you to _Victor Stinner's Guide_ (which I believe is this: https://vstinner.readthedocs.io/) and other broken links in that document should be fixed

## Screenshots
s1
s2
s3

### PR DESCRIPTION
**Pull request by Dev-satish**
Fixes #1044

## Changes Made 

1.   In ``advanced-tools/gdb.rst`` there was a broken link under the topic of [GDB 7 and later](https://devguide.python.org/advanced-tools/gdb/#gdb-7-and-later) which was named as _extending gdb with Python_ ``https://sourceware.org/gdb/current/onlinedocs/gdb/Python.html#Python`` which has now been replaced by the following link ``https://sourceware.org/gdb/onlinedocs/gdb/Python.html``

2. In ``conf.py`` there was a _slash_  at the end of the link ``https://github.com/python/voters/`` which made this link broken and this issue has been resolved by removing the _slash_ from the end of the link ``https://github.com/python/voters``

### CODE REVIEW

File: advanced-tools/gdb.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ GDB 7 and later
 ===============
 
 In gdb 7, support for `extending gdb with Python
-<https://sourceware.org/gdb/current/onlinedocs/gdb/Python.html#Python>`_ was
+<https://sourceware.org/gdb/onlinedocs/gdb/Python.html#Python>`_ was
```
Comment (hugovk) [inline advanced-tools/gdb.rst:24]: We don't really need the `#Python` bit. There is an anchor on the page, but it's right at the top.

```suggestion
<https://sourceware.org/gdb/onlinedocs/gdb/Python.html>`_ was
```

**Review by hugovk — COMMENTED** [review]
One little suggestion.

And please could you edit your PR message at the top here and add in "Fixes #1044"?

Then, when this PR is merged, it will automatically close that issue.

Thank you!

**Comment by CAM-Gerlach** [comment]
> File : conf.py
> line changed : 84
>
> File : advanced-tools/gdb.rst
> line changed : 24

Just a tip—instead of repeating a summary of the information that can be easily seen in much more detail by clicking the `Files Changed` tab, it would be much more useful to provide a brief overall summary of your meaningful changes.

> And please could you edit your PR message at the top here and add in "Fixes https://github.com/python/devguide/issues/1044"?

Just FYI, core team members (devs and now triagers) can edit posts, and you can also add it directly under `Development` in the sidebar. My only hesitation there is that we might not want to close that issue until the relevant information is incorporated into the devguide itself, since this is only a stopgap fix.

**Comment by darechan** [comment]
> > File : conf.py
> > line changed : 84
> > File : advanced-tools/gdb.rst
> > line changed : 24
> 
> Just a tip—instead of repeating a summary of the information that can be easily seen in much more detail by clicking the `Files Changed` tab, it would be much more useful to provide a brief overall summary of your meaningful changes.
> 
> > And please could you edit your PR message at the top here and add in "Fixes #1044"?
> 
> Just FYI, core team members (devs and now triagers) can edit posts, and you can also add it directly under `Development` in the sidebar. My only hesitation there is that we might not want to close that issue until the relevant information is incorporated into the devguide itself, since this is only a stopgap fix.

Understood, thanks for the feedback! I'll change it right away

File: advanced-tools/gdb.rst (hunk 1 above)
Comment (Dev-satish) [reply advanced-tools/gdb.rst:24]: understood, thanks for the feedback I'll change it right away

**Comment by Dev-satish** [comment]
> > File : conf.py
> > line changed : 84
> >
> > File : advanced-tools/gdb.rst
> > line changed : 24
> 
> Just a tip—instead of repeating a summary of the information that can be easily seen in much more detail by clicking the `Files Changed` tab, it would be much more useful to provide a brief overall summary of your meaningful changes.
> 
> > And please could you edit your PR message at the top here and add in "Fixes https://github.com/python/devguide/issues/1044"?
> 
> Just FYI, core team members (devs and now triagers) can edit posts, and you can also add it directly under `Development` in the sidebar. My only hesitation there is that we might not want to close that issue until the relevant information is incorporated into the devguide itself, since this is only a stopgap fix.



Reply understood, thanks for the feedback I'll change it right away

**Comment by Dev-satish** [comment]
@CAM-Gerlach, can you check if the documentation is correct ?

**Review by hugovk — APPROVED** [review]
The linkcheck on the CI is all green now: https://github.com/python/devguide/actions/runs/4150295949/jobs/7179852698

Thank you again for your contribution!

**Comment by Dev-satish** [comment]
> The linkcheck on the CI is all green now: https://github.com/python/devguide/actions/runs/4150295949/jobs/7179852698
> 
> Thank you again for your contribution!

Thank you so much for giving this opportunity to me! If there are any other similar issues or issues bit higher than this level for me then please let me know as I want to learn further and understand how contribution works. Even resources to contribute in python or tips for future work and learning would be a great asset to me!

**Comment by CAM-Gerlach** [comment]
> @CAM-Gerlach, can you check if the documentation is correct ?

Assuming you mean your description in the OP, I just meant that to be taken as a helpful tip for the future rather than anything required now, but you did an excellent job!

Aside from simple devguide, docs, etc. fixes like this, IMO one good place to start contributing (and where I started myself in open source originally, before I even knew that much more than a beginner level of Python) is helping triage new issues. If the issue is a bug, try to reproduce it and report your results, or request additional required/clarifying information from the OP if not. If it appears to be a user-side issue, give the user troubleshooting advice and redirect them to a more appropriate forum (e.g. Python Help on the Python Discourse). If its a feature request, ensure it is complete, valid and well-specified, etc. See [the devguide pages on triaging](https://devguide.python.org/triage/triaging/) for more guidance.

Beyond that, you can also:

* [find and fix "easy" issues issues](https://devguide.python.org/getting-started/fixing-issues/).
* [help with a translation](https://devguide.python.org/documentation/translating/). 
* Review pull requests that add new documentation to ensure it is clear and straightforward to follow for beginners

**Pull request merged**